cinema_service/settings.py

```diff
@@ -125,13 +125,15 @@
 
 USE_I18N = True
 
-USE_TZ = False
+USE_TZ = True
 
 
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/4.0/howto/static-files/
 
 STATIC_URL = "static/"
+MEDIA_URL = "/media/"
+MEDIA_ROOT = BASE_DIR / "media"
 
```

Comment:
you should ignore media files, so let's remove them(because it's your local fiels and we don't need them here) + ignore in the future commits